### PR TITLE
for the two recent commits

### PR DESCRIPTION
**Pull request by freliciatucker**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,74 @@
...
+            <a href="https://github.com/freliciatucker">GitHub profile</a> 
+            or look at my 
+            <a href="https://www.linkedin.com/in/frelicia-t-3444bb146/">
+            LinkedIn page</a> to research other things 
+            I'm doing to grow professionally</p>
+
+        <h4> Fun Facts! </h4>
+          <p>Click below to learn a random fact about me!</p>
+          <button><a href="randomFact.html">Click Me!</a></button>
+          <p> <a href="http://jigsaw.w3.org/css-validator/check/referer"> <img style="border:0;width:10px;height:11px"
```
Comment (jmchristen) [inline portfolio/src/main/webapp/index.html:69]: Pull the styling out into CSS.

https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# they tell me what to type
+
```
Comment (jmchristen) [inline readme.md:2]: I think you don't need to add this file.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -1,28 +1,50 @@
...
+      'Tragically he was killed only 2 years after being released, but I ' +
+      'continue to remember his legacy by learning computer science and ' +
+      'staying physically active today.']
+    
+    // Match picture number to string index.
+    const caption = captions[num];
+
+   // Add string to the page.
+   const hiddenPrint =  document.getElementById('img' + num + '-container');
+   console.log(hisddenPrint);
```
Comment (jmchristen) [inline portfolio/src/main/webapp/script.js:46]: Typo here, and you can probably remove the log line anyway. This kind of log is generally only useful while debugging.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (freliciatucker) [reply portfolio/src/main/webapp/index.html:69]: deleted this section for now

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -1,28 +1,50 @@
...
+   hiddenPrint.innerText = caption; 
+
+   return;
```
Comment (zjwoody) [inline portfolio/src/main/webapp/script.js:49]: You don't need the return at the end of the void function.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -1,28 +1,50 @@
...
-  greetingContainer.innerText = greeting;
+  const factsContainer = document.getElementById('fact-container');
+  factsContainer.innerText = fact;
 }
+
+/**
+* Opens picture up to text of the associated paragraph.
+ */
+function openPicture(num){
+
```
Comment (zjwoody) [inline portfolio/src/main/webapp/script.js:23]: Unnecessary blank line.

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -1,18 +1,74 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <h1>Frelicia Tucker's Portfolio</h1>
+    
+    <button onclick="openPicture(0)"><img src="images/freliciaface.jpg" alt="Frelicia's headshot"/></button>
```
Comment (zjwoody) [inline portfolio/src/main/webapp/index.html:14]: Seems you already have a image link in button, so the image will show no matter you click or not, I think you can remove the \<img and try to only show the image in openPicture onclick.

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -18,13 +18,37 @@
 function addRandomFact() {
   const facts =
       ['I have a day named after me', 'I have a dog named Romeo', 
-      'My favorite movie is Les Mis', 'The first word I could spell was "cow"'];
+      'My favorite movie is Les PMis', 'The first word I could spell was "cow"'];
 
-  // Pick a random fact.
+  //Pick a random fact.
```
Comment (zjwoody) [inline portfolio/src/main/webapp/script.js]: Need a blank between // and first word. Comment starts with capital and ends with period, same as other comments.
You can run a clang-format to fix the code format.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jmchristen) [reply portfolio/src/main/webapp/index.html:69]: It's still there in the snapshot I see.

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (freliciatucker) [reply portfolio/src/main/webapp/index.html:14]: the openPicture function probably has a confusing name. it just pairs the text to the picture, it doesnt show the image

Comment (zjwoody) [reply portfolio/src/main/webapp/index.html:14]: Maybe you can rename the function to a more clear name, e.g. showPictureText() etc.

**Pull request merged**